README.md

```diff
@@ -145,6 +147,42 @@ pk.search('Paris', {
 | `query` | `string` | Search terms |
 | `opts` | `key-value mapping` (optional) | Search-specific parameters (see [options](#pkoptions)) |
 
+### `pk.reverse()`
+
+Performs a reverse geocoding search and returns a Promise, which response is a list of results alongside some request metadata.
+The options passed as second parameter override the global parameters only for the current query.
+Any `coordinates` previously set as option would be overriden by the coordinates passed as first argument.
+
+```js
+pk.reverse('48.871086,2.3036339', {
```

Comment:
The API allows empty `coordinates`. Not sure if it makes sense with the client UI but it's doable. It's less accurate and fallback on the IP located